firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -83,4 +86,15 @@ private static String encodeFidBase64UrlSafe(byte[] rawValue) {
     bb.putLong(uuid.getLeastSignificantBits());
     return bb.array();
   }
+
+  public static long getCurrentTimeInSeconds() {
```

Comment:
imo your testing life is going to be way easier if you make these non-static and inject an instance of Utils. For time stuff there's already com.google.android.gms.common.util.Clock and  com.google.android.gms.common.util.DefaultClock that would be a better choice